PublicGitArchive/pga/cmd/siva.go

```diff
@@ -221,7 +221,7 @@ func dumpFiles(commit *object.Commit, destDir string) error {
 	return err
 }
 
-// sivaCmd represents the list command
+// sivaCmd represents the siva command
```

Comment:
I wrote a pretty bad comment :D

```suggestion
// sivaCmd represents the set of commands to work with the siva files: extract revisions, list, dump raw contents.
```

Reply:
:100: